packages/action-menu/src/ActionMenu.ts

```diff
@@ -43,6 +43,9 @@ export class ActionMenu extends ObserveSlotText(PickerBase, 'label') {
     @property({ type: String })
     public override selects: undefined | 'single' = undefined;
 
+    @property({ type: String, reflect: true })
+    public static: 'white' | 'black' | 'none' = 'none';
```

Comment:
This will reflect `static="none"` at runtime, which has negative performance implications. We should use `undefined` as the default. 

@Rajdeepc makes a good point about Storybook, but we can manage the experience delivered in that context without needing to surface `'none'` as an acceptable value for this property.

Reply:
OK!